src/plugins/augeas/README.md

```diff
@@ -45,6 +45,12 @@ following command:
 kdb mount /etc/hosts system:/hosts augeas lens=Hosts.lns
 ```
 
+And to unmount the file run:
+
+```sh
+kdb umount system:/hosts
```

Comment:
sudo should be always added to `kdb umount` as normally you need it (if you didn't do one of the tricks of doc/TESTING.md)

```suggestion
sudo kdb umount system:/hosts
```